guide/java/policies.md

```diff
@@ -235,6 +235,30 @@ An Enricher which computes latency in accessing a URL.
 Can be used to combine the values of sensors.  This enricher should be instantiated using Enrichers.buider.combining(..).
 This enricher is only available in Java blueprints and cannot be used in YAML.
 
+#### Note On Enricher Producers
+
+`enricher.producer: brooklyn:entity("entity.id.here")` is a `brooklyn.config` key/value pair that should be supplied to any
```

Comment:
Thanks a lot -- I was struggling to write that paragraph in a clear way and felt as thought it was a tough read as well.